app/services/websocket.py

```
@@ -22,13 +31,102 @@ async def connect(self, websocket: WebSocket, user_id: str):
+            message = {
+                "type": "market_update",
+                "symbol": symbol,
+                "data": price_data,
+                "timestamp": price_data.get("timestamp")
+            }
+            
+            broken_connections = []
+            for connection in self.market_subscribers[symbol]:
+                try:
+                    await connection.send_json(message)
+                except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)